keyboards/gopolar/gg86/info.json

```
@@ -0,0 +1,295 @@
+                {"label":"Q", "x":1.5, "y":2.5},
+                {"label":"W", "x":2.5, "y":2.5},
+                {"label":"E", "x":3.5, "y":2.5},
+                {"label":"R", "x":4.5, "y":2.5},
+                {"label":"T", "x":5.5, "y":2.5},
+                {"label":"Y", "x":6.5, "y":2.5},
+                {"label":"U", "x":7.5, "y":2.5},
+                {"label":"I", "x":8.5, "y":2.5},
+                {"label":"O", "x":9.5, "y":2.5},
+                {"label":"P", "x":10.5, "y":2.5},
+                {"label":"[{", "x":11.5, "y":2.5},
+                {"label":"]}", "x":12.5, "y":2.5},
+                {"label":"\\| / ISO Hash", "x":13.5, "y":2.5, "w":1.5},
+                {"label":"Delete", "x":15.25, "y":2.5},
+                {"label":"End", "x":16.25, "y":2.5},
+                {"label":"PgDn", "x":17.25, "y":2.5},
+
+                {"label":"Caps Lock", "x":0, "y":3.5, "w":1.75},
+                {"label":"A", "x":1.75, "y":3.5},
+                {"label":"S", "x":2.75, "y":3.5},
+                {"label":"D", "x":3.75, "y":3.5},
+                {"label":"F", "x":4.75, "y":3.5},
+                {"label":"G", "x":5.75, "y":3.5},
+                {"label":"H", "x":6.75, "y":3.5},
+                {"label":"J", "x":7.75, "y":3.5},
+                {"label":"K", "x":8.75, "y":3.5},
+                {"label":"L", "x":9.75, "y":3.5},
+                {"label":";:", "x":10.75, "y":3.5},
+                {"label":"'\"", "x":11.75, "y":3.5},
+                {"label":"Enter", "x":12.75, "y":3.5, "w":2.25},
+
+                {"label":"Shift", "x":0, "y":4.5, "w":1.25},
+                {"label":"ISO \\", "x":1.25, "y":4.5},
+                {"label":"Z", "x":2.25, "y":4.5},
+                {"label":"X", "x":3.25, "y":4.5},
+                {"label":"C", "x":4.25, "y":4.5},
+                {"label":"V", "x":5.25, "y":4.5},
+                {"label":"B", "x":6.25, "y":4.5},
+                {"label":"N", "x":7.25, "y":4.5},
+                {"label":"M", "x":8.25, "y":4.5},
+                {"label":",<", "x":9.25, "y":4.5},
+                {"label":".>", "x":10.25, "y":4.5},
+                {"label":"/?", "x":11.25, "y":4.5},
+                {"label":"Shift", "x":12.25, "y":4.5, "w":1.75},
+                {"label":"Shift", "x":14, "y":4.5},
+                {"label":"\u2191", "x":16.25, "y":4.5},
+
+                {"label":"Ctrl", "x":0, "y":5.5, "w":1.5},
+                {"label":"Win", "x":1.5, "y":5.5},
+                {"label":"Alt", "x":2.5, "y":5.5, "w":1.5},
+                {"label":"Space", "x":4, "y":5.5, "w":7},
+                {"label":"Alt", "x":11, "y":5.5, "w":1.5},
+                {"label":"Fn", "x":12.5, "y":5.5},
+                {"label":"Ctrl", "x":13.5, "y":5.5, "w":1.5},
+                {"label":"\u2190", "x":15.25, "y":5.5},
+                {"label":"\u2193", "x":16.25, "y":5.5},
+                {"label":"\u2192", "x":17.25, "y":5.5}
+            ]
+        }
+        "LAYOUT_ansi_tsangan": {
```

Needs to match the layout name used in `gg86.h`.

```suggestion
        "LAYOUT_tkl_ansi_tsangan": {
```